src/io/flutter/module/InstallSdkAction.java

```diff
@@ -143,6 +145,7 @@ public void validateSelectedFiles(VirtualFile[] files) throws Exception {
 
       final VirtualFile installTarget = FileChooser.chooseFile(descriptor, null, null);
       if (installTarget != null) {
+        FlutterInitializer.sendAnalyticsAction(getClass().getSimpleName());
```

Comment:
On mobile now, but perhaps we want separate analytics calls for each case then - 'InstallSdkAction' and 'ShowInstallDocsAction'?